scripts/generate_afrisenti_benchmark.sh

```diff
@@ -3,38 +3,47 @@ model_path=configs/models
 batch_size=8
 
 declare -a models
-export CUDA_VISIBLE_DEVICES=1,2,3,4,5,6,7
+export CUDA_VISIBLE_DEVICES=0,1,2
 
 models=(
     
 )
 
 models=(
-    # "afriteva_v2_large_ayaft"
-    # "meta_llama_8b_instruct"
-    # "aya_101"
-    "bigscience_bloomz_1b7"
+    "bigscience_mt0_small"
+    "bigscience_mt0_xxl"
+    "google_gemma-1_7b_it"
+    "meta_llama-2_7b_chat"
+    "google_gemma-2_9b_it"
+    "meta_llama_8b_instruct"
+    "aya_101"
+    "google_gemma-2_27b_it"
+    "meta_llama_70b_instruct"
+    "afriteva_v2_large_ayaft"
+    "jacaranda_afrollama" 
+    # "bigscience_bloomz_1b7"
     # "bigscience_bloomz_3b"
     # "bigscience_bloomz_7b1"
     # "lelapa_inkuba_0_4b"
     # "bigscience_mt0_base"
-    # "bigscience_mt0_large"
-    # "bigscience_mt0_small"
+    # "bigscience_mt0_large" 
     # "bigscience_mt0_xl"
     # "google_flan_t5_base"
     # "google_flan_t5_large"
     # "google_flan_t5_small"
 )
 
-for num_fewshot_samples in 0 5
-do 
-    for model in "${models[@]}"
-    do
-        echo "Running model: $model"
-
+for model in "${models[@]}"
+do
+  echo "Evaluating model: $model"
+  for fewshot in 0 5
+  do
         python src/evaluate.py --model-config-yaml ${model_path}/${model}.yaml \
         --task-config-yaml ${task_config} \
-        --eval.num-fewshot ${num_fewshot_samples} \
-        --eval.batch-size ${batch_size}
+        --eval.num-fewshot $fewshot \
+        --eval.batch-size ${batch_size} \
+        --eval.device 'cuda:0' \
```

Comment:
If you already set line 6 there's no need to set the device here again

Reply:
was it running when you added this?

Reply:
We need to figure out though, because with that parameter you won't be able to run a model that doesn't fit into one GPU